homeassistant/components/avea/light.py

```diff
@@ -30,32 +30,12 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class AveaLight(LightEntity):
     """Representation of an Avea."""
 
+    _attr_supported_features = SUPPORT_AVEA
+
     def __init__(self, light):
         """Initialize an AveaLight."""
         self._light = light
-        self._name = light.name
-        self._state = None
-        self._brightness = light.brightness
```

Comment:
Removing this line changes the behavior of the integration, is that expected?

Reply:
In line 27, should we add `, True` and then remove brightness from init?